src/main/java/frc/robot/vision/photonvision/photonvisionfilters/PhotonVisionAprilTagFilters.java

```diff
@@ -0,0 +1,49 @@
+package frc.robot.vision.photonvision.photonvisionfilters;
+
+import edu.wpi.first.math.geometry.Pose2d;
+import frc.robot.poseestimator.PoseEstimationMath;
+import frc.robot.poseestimator.observations.VisionObservation;
+import frc.robot.vision.VisionRawData;
+import frc.robot.vision.aprilTags.AprilTagFilters;
+import frc.robot.vision.aprilTags.AprilTagFiltersTolerances;
+import frc.robot.vision.photonvision.CameraConfiguration;
+import frc.utils.Conversions;
+import org.littletonrobotics.junction.Logger;
+
+import java.util.ArrayList;
+import java.util.function.Function;
+
+public class PhotonVisionAprilTagFilters extends PhotonVisionFiltered {
+
+	private final Function<Double, Pose2d> getEstimatedPoseAtTimestamp;
+	private final AprilTagFiltersTolerances tolerances;
+
+	public PhotonVisionAprilTagFilters(
+		CameraConfiguration[] cameraConfigurations,
+		String logPath,
+		Function<Double, Pose2d> getEstimatedPoseAtTimestamp,
+		AprilTagFiltersTolerances tolerances
+	) {
+		super(cameraConfigurations, logPath);
+		this.getEstimatedPoseAtTimestamp = getEstimatedPoseAtTimestamp;
+		this.tolerances = tolerances;
+	}
+
+	@Override
+	protected boolean keepPhotonVisionData(VisionRawData visionData) {
+		return AprilTagFilters.keepRobotPoseData(visionData, calculateCurrentPoseEstimation(), tolerances);
```

Comment:
the photon observations should not be filtered using the filters used on the limelight observations

Reply:
these are the same filters